### ISSUE TITLE
unknown package 

### ISSUE THREAD

**Issue by faisalsafar99** [OHDSI/Achilles#647]
i have tried to run the below commands in r Studio: 
install.packages("remotes")
remotes::install_github("OHDSI/Achilles")

then i got the below error message. 

Error: Failed to install 'unknown package' from GitHub:
  Timeout was reached: [api.github.com] Resolving timed out after 10000 milliseconds

May you help me please?

**Comment by fdefalco** [comment]
This looks like a timeout issue, not specific the achilles package.  I've been able to successfully run the install from a few different environments.   Suggest trying installation again.

**Comment by faisalsafar99** [comment]
> This looks like a timeout issue, not specific the achilles package. I've been able to successfully run the install from a few different environments. Suggest trying installation again.

Thank you in advance for your replay and help.

I have tried multiple ways as below


**> install.packages("https://github.com/OHDSI/Achilles.git", repo=NULL, type="source")**
Installing package into ‘C:/Users/Faisal/Documents/R/win-library/4.1’
(as ‘lib’ is unspecified)
trying URL 'https://github.com/OHDSI/Achilles.git'
Content type 'text/html; charset=utf-8' length unknown
downloaded 186 KB

Error in getOctD(x, offset, len) : invalid octal digit
Warning message:
In install.packages("https://github.com/OHDSI/Achilles.git", repo = NULL,  :
  installation of package ‘C:/Users/Faisal~1/AppData/Local/Temp/RtmpukePdU/downloaded_packages/Achilles.git’ had non-zero exit status
> 


----------------------------------------------------------------

**> install.packages("C:/Users/Faisal/Downloads/Achilles-main/Achilles-main")**

WARNING: Rtools is required to build R packages but is not currently installed. Please download and install the appropriate version of Rtools before proceeding:

https://cran.rstudio.com/bin/windows/Rtools/
Installing package into ‘C:/Users/Faisal/Documents/R/win-library/4.1’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘C:/Users/Faisal/Downloads/Achilles-main/Achilles-main’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages
> 

--------------------------------------------------------------------------------------------------

**> install.packages("remotes")**
WARNING: Rtools is required to build R packages but is not currently installed. Please download and install the appropriate version of Rtools before proceeding:

https://cran.rstudio.com/bin/windows/Rtools/
Installing package into ‘C:/Users/Faisal/Documents/R/win-library/4.1’
(as ‘lib’ is unspecified)
trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.1/remotes_2.4.2.zip'
Content type 'application/zip' length 399478 bytes (390 KB)
downloaded 390 KB

package ‘remotes’ successfully unpacked and MD5 sums checked

The downloaded binary packages are in
	C:\Users\fsmutairi\AppData\Local\Temp\RtmpyGUvP6\downloaded_packages
**> remotes::install_github("OHDSI/Achilles")**
Error: Failed to install 'unknown package' from GitHub:
  Timeout was reached: [api.github.com] Resolving timed out after 10000 milliseconds




**Here is about my RStudio :**

RStudio 2022.02.1+461 "Prairie Trillium" Release (8aa5d470d82d615130db663ace57992d4e3, 2022-03-17) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko)

**Comment by faisalsafar99** [comment]
> Appears you need to install RTools. Could you try following the environment setup steps outlined here:
> 
> https://ohdsi.github.io/Hades/rSetup.html

yes, I downloaded rSetup and I followed the steps there as it is.
 Also, I tried to download RTools and I got the same issue 
I still faced the issues and I think the issue is the proxy. 

I would like to have someone, he is able to help me and connect with my machine remotely to figure out the issue.

Thank you in advance Mr. DeFalco, Frank
